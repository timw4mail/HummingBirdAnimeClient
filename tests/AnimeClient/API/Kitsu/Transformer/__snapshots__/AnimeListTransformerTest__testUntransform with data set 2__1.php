<?php return Aviat\AnimeClient\Types\FormItem::__set_state(array(
   'id' => 14047983,
   'anilist_item_id' => NULL,
   'mal_id' => '12347',
   'data' => 
  Aviat\AnimeClient\Types\FormItemData::__set_state(array(
     'notes' => '',
     'private' => true,
     'progress' => 12,
     'reconsumeCount' => 0,
     'reconsuming' => true,
     'status' => 'current',
  )),
));
