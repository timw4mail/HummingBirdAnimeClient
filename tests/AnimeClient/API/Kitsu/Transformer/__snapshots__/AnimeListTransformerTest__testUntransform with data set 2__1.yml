empty: false
id: 14047983
mal_id: '12347'
data:
    empty: false
    notes: ''
    private: true
    progress: 12
    ratingTwenty: null
    reconsumeCount: 0
    reconsuming: true
    status: current
